City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3818 Vocational school search

How to test:

1. Set up your local site with this branch.
2. `make drush-cim && make drush-cr`
3. `make shell`
4. `drush locale-import fi /app/public/themes/custom/hdbt_subtheme/translations/fi.po`
5. `drush cr`
6. Login to the site and make sure you have new paragraph in your disposal called Vocational school search. It should be available in basic pages and landing pages.
7. You can reference TPR-units to the paragraph and it should look similar to High school search paragraph with the distinction that there is no filters at this point. The design is here: https://app.abstract.com/share/78d5fe8f-6d31-443d-9cd3-ebfd51fad63a?collectionLayerId=4f46a1df-1244-42c8-8503-26a274b70880&mode=design
8. Check that the block works well on desktop and mobile. Also make sure that High school search block works as it did before with its High school front page functionality because the backend functionality had some changes that touched both high school and vocational school cards.
